src/e2e/java/teammates/e2e/cases/lnp/FeedbackSessionSubmitLNPTest.java

```diff
@@ -252,11 +256,19 @@ protected ListedHashTree getLnpTestPlan() {
         return testPlan;
     }
 
+    @Override
+    protected void setupSpecification() {
+        this.specification = LNPSpecification.builder()
+                                             .withErrorRateLimit(ERROR_RATE_LIMIT)
+                                             .withMeanRespTimeLimit(MEAN_RESP_TIME_LIMIT)
+                                             .build();
```

Comment:
Try not to have this kind of custom indentation. The intention may be good (e.g. to align with the dots at the previous line), but if any of the previous line changes (e.g. the `LNPSpecification` class name's change), these lines will be affected as well.

It should just be simple 8-space indentation, i.e. 2 steps (where 1 step is 4-space indentation).
```java
        this.specification = LNPSpecification.builder()
                .withErrorRateLimit(ERROR_RATE_LIMIT)
                .withMeanRespTimeLimit(MEAN_RESP_TIME_LIMIT)
                .build();
```

Reply:
Thanks for the advice! All styles are updated as suggested.